apps/ycrv/components/list/GaugeList.tsx

```diff
@@ -99,6 +109,10 @@ function GaugeList({gauges, gaugesVotes, isLoading, userInfo}: TProps): ReactEle
 	const maxVotes = userInfo.balance.sub(userInfo.votesSpent);
 	const [{votes, ...votesState}, votesDispatch] = useReducer(votesReducer, {votes: {}, maxVotes}, createInitialState);
 
+	useUpdateEffect((): void => {
+		votesDispatch({type: 'INIT_MAX', maxVotes});
+	}, [formatBN(maxVotes).toString()]);
+
```

Comment:
As mentioned above, it would be cleaner to move this `const maxVotes = userInfo.balance.sub(userInfo.votesSpent);` to the reducer, instead of doing it here